.env

```diff
@@ -117,13 +116,18 @@ MF_PROVISION_THINGS_LOCATION=http://things:8182
 MF_PROVISION_USER=
 MF_PROVISION_PASS=
 MF_PROVISION_API_KEY=
-MF_PROVISION_CERTS_SVC_URL=http://localhost/certs
+MF_PROVISION_CERTS_SVC_URL=
 MF_PROVISION_X509_PROVISIONING=false
 MF_PROVISION_BS_SVC_URL=http://bootstrap:8202/things/configs
 MF_PROVISION_BS_SVC_WHITELIST_URL=http://bootstrap:8202/things/state
 MF_PROVISION_BS_CONFIG_PROVISIONING=true
 MF_PROVISION_BS_AUTO_WHITELIST=true
 MF_PROVISION_BS_CONTENT=
```

Comment:
Why all this envars are empty? I would prefer to copy paste values from main here